packages/configuration-validation/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@okta/configuration-validation",
+  "version": "1.0.0",
```

Comment:
Should we start from 0.0.1 like in the jwt-verifier?

Reply:
We can probably start with an `0.1.0` version, if we want to offer this as a pre-release.

I have a strong feeling that developers are **not** going to consume this library directly, deferring to our integrations.